intro_to_circleci.md

```diff
@@ -459,19 +445,10 @@ By default, CircleCI builds every commit from every branch. This behavior may be
 
 **Note:** Even if this option is enabled, CircleCI will still build all commits from your project’s default branch.
 
-### Build Pull Requests From Forked Repositories
-Many open source projects accept PRs from forked repositories. Building these PRs is an effective way to catch bugs before manually reviewing changes.
-
-By default, CircleCI does not build PRs from forked repositories. To change this setting, go to the **Advanced Settings** of your project and set the **Build forked pull requests** option to On.
-
-### Caching
-Caches are isolated based on GitHub Repo for PRs. CircleCI uses the GitHub repository-id of the originator of the fork PR to identify the cache.
-
-- PRs from the same fork repo will share a cache (this includes, as previously stated, that PRs in the master repo share a cache with master).
-
-- Two PRs in different Fork Repos will have different caches.
+### Auto-cancel redundant builds
+With the exception of your default branch, we will automatically cancel any queued or running builds on a branch when a newer build is triggered on that same branch. Scheduled workflows and re-runs are not auto-canceled.
```

Comment:
Not sure about "we". Sounds strange. I would change it to this one:

With the exception of your default branch, any queued or running builds on a branch will be automatically cancel when a newer build is triggered on that same branch. Scheduled workflows and re-runs are not auto-canceled.